bothub/utils.py

```diff
@@ -89,6 +92,30 @@ def unique_slug_generator(validated_data, Repository, new_slug=None):
     return slug
 
 
+def organization_unique_slug_generator(org_name, RepositoryOwner, new_slug=None):
+    """
+    This is for a Django project and it assumes your instance
+    has a model with a slug field and a title character (char) field.
+    """
+    if new_slug is not None:
+        slug = new_slug
+    else:
+        slug = slugify(org_name)[:9:]
+
+    qs_exists = RepositoryOwner.objects.filter(nickname=slug).exists()
+    if qs_exists:
```

Comment:
it is more common, in python, to rely on EAFP (It’s easier to ask for forgiveness than permission) than on LBYL (Look before you leap)
https://docs.python.org/3.5/glossary.html#term-eafp